spec/searchComponentSpec.js

```diff
@@ -584,33 +587,52 @@ describe('gameSearchCtrl',function(){
       * Adapted from: 
       * http://stackoverflow.com/questions/4434076/best-way-to-alphanumeric-check-in-javascript
       */
-        isAlphaNumeric = function(str){
-      // a to function check if the user only entered letters or numbers in their search query
-      var code, i, len;
       
-      for( i = 0, len = str.length; i < len; i++){
-        code = str.charCodeAt(i);
-        console.log(code);
-        if(!(code > 47 && code < 58) && //numeric
-        !(code > 64 && code < 91 ) && // uppercase letters
-        !(code > 96 && code < 123) && // lowercase letters
-        // foreign language characters
-        !(code >= 128 && code <= 155) && 
-        !(code == 157) && 
-        !(code >= 160 && code <= 165) &&
-        !(code >= 181 && code <= 183) &&
-        !(code == 198 && code == 199) &&
-        !(code >= 208 && code <= 212) &&
-        !(code >= 214 && code <= 216) &&
-        !(code >= 224 && code <= 229) &&
-        !(code >= 233 && code <= 237) &&
-        !(code == 32)){
+    
+      altNameChecker = function(){
```

Comment:
You should not have this code in your spec file. Instead, you should test the code in gameapp.js at line 39